packages/design-system/src/components/HelpDrawer/HelpDrawerToggle.jsx

```diff
@@ -14,7 +14,7 @@ export class HelpDrawerToggle extends React.PureComponent {
   }
 
   render() {
-    const { className, children, inline, showDrawer, ...others } = this.props;
```

Comment:
This is something we forgot to do in the past, but we should also update `HelpDrawerToggle.d.ts` to use the React.HTMLProps for the button element considering we allow props to be passed down.

Something like this: 
```
export default class HelpDrawerToggle extends React.Component<
React.HTMLProps<HTMLButtonElement> & HelpDrawerToggleProps,
  any
> {
  render(): JSX.Element;
}
```

Reply:
@nichia or I can also handle this if you are busy @erinrep, just let us know!

Reply:
@bernardwang that would be great, I've got a lot on my plate today. Thanks!